pages/landing/patcher/how-it-works/_maintainer-how-it-works.html

```diff
@@ -1,11 +1,11 @@
 {% for item in page.maintainer_how_it_works %}
   <div class="row">
     {% if item.precode %}
-      <div class="item_precode row-image-content {% if item.image_side == "left" %}col-xs-12 col-sm-7 col-md-6 col-description{% else %}col-xs-12 col-sm-7 col-sm-pull-5 col-md-6 col-md-pull-6 col-description{% endif %}">
+      <div class="item_precode row-image-content {% if item.image_side == "left" %}col-xs-8 col-xs-offset-2 col-sm-5 col-sm-offset-0 col-md-6 col-diagram hidden-print{% else %}col-xs-10 col-xs-offset-1 col-sm-5 col-sm-push-7 col-sm-offset-0 col-md-6 col-md-push-6 col-diagram{% endif %}">
         {{ item.precode }}
       </div>
     {% endif %}
-        {% if item.tab_block %}
+    {% if item.tab_block %}
```

Comment:
The changes here and below are purely indentation fixes. It was hard to read the code otherwise.